### DISCUSSION TITLE
I try to install tabby in neovim but...

### DISCUSSION THREAD

**Discussion by fjmarcelo** [nanozuki/tabby.nvim#101] [Q&A] [answered]
... when I use vim-plug, I have this message

**fatal: invalid reference: master**

Any suggestion?

Thanks in advance.

**Comment by laxect** [comment, marked as answer]
Try use `{'branch': 'main'}` ?

**Comment by fjmarcelo** [comment]
Thank you very much!! It works!


El dom, 18 dic 2022 a las 9:20, ギャラ ***@***.***>) escribió:

> Try use {'branch': 'main'} ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nanozuki/tabby.nvim/discussions/101#discussioncomment-4440972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOALU43X7UHBU6G3532TCTWN3JMHANCNFSM6AAAAAATBS2LNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Félix J. Marcelo Wirnitzer *

Este mensaje de correo electrónico y sus documentos adjuntos están
dirigidos EXCLUSIVAMENTE a los destinatarios especificados. La información
contenida puede ser CONFIDENCIAL y/o estar LEGALMENTE PROTEGIDA. Si usted
recibe este mensaje por ERROR, por favor comuníquelo inmediatamente al
remitente y ELIMÍNELO ya que usted NO ESTA AUTORIZADO al uso, revelación,
distribución, impresión o copia de toda o alguna parte de la información
contenida.
Gracias.
--
This e-mail message and its attachments are addressed EXCLUSIVELY to the
specified recipients. The information contained herein may be CONFIDENTIAL
and/or LEGALLY PROTECTED. If you receive this message in ERROR, please
notify the sender immediately and REMOVE IT as you ARE NOT AUTHORISED to
use, disclose, distribute, print or copy all or any part of the information
contained herein.
Thank you.
--
Diese E-Mail-Nachricht und ihre Anlagen sind AUSSCHLIESSLICH an die
angegebenen Empfänger gerichtet. Die hierin enthaltenen Informationen
können VERTRAULICH und/oder GESETZLICH GESCHÜTZT sein. Wenn Sie diese
Nachricht als FEHLER erhalten, benachrichtigen Sie bitte sofort den
Absender und ENTFERNEN Sie sie, da Sie NICHT befugt sind, die hierin
enthaltenen Informationen ganz oder teilweise zu verwenden, weiterzugeben,
zu verbreiten, zu drucken oder zu kopieren.
Ich danke Ihnen.
